app/views/submissions/_submission_tile.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="tile<% unless submission.content.children.empty? %> tile-stack<% end %>" data-submission-id="<%= submission.id %>">
+<div class="tile<% if !submission.content.nil? && !submission.content.children.empty? %> tile-stack<% end %>" data-submission-id="<%= submission.id %>">
   <%= link_to [submission.feed, submission], class: "tile-content" do %>
     <%= render_content(submission.content, {type: 'tile'}) %>
```

Comment:
You're only affecting the class so the "stacked" icon does not appear if there are no children (or no content).  I'd think you'd want to make that render conditional also (if the content was not nil), at the least, or it will blow up there next.